client/src/app-components/ticket-list.js

```diff
@@ -21,7 +22,10 @@ class TicketList extends React.Component {
         type: React.PropTypes.oneOf([
             'primary',
             'secondary'
-        ])
+        ]),
+        showClosedTickets: React.PropTypes.bool,
```

Comment:
What do you think of changing the name to`closedTicketsShown`? since this component doesn't filter them